setup.py

```diff
@@ -82,13 +82,19 @@ def get_version(version_file):
     ],
     python_requires=">=3.6",
     # What does your project relate to?
-    keywords="skeleton",
+    keywords="li-pca",
```

Comment:
Feel free to add some more useful keywords here, e.g.:
```suggestion
    keywords="api li-pca pca phishing",
```